src/service.ts

```diff
@@ -328,6 +329,20 @@ export class DebugService implements IDebugger {
     return reply.body.scopes;
   };
 
+  getVariable = async (_: any, variable: DebugProtocol.Variable) => {
+    const reply = await this.session.sendRequest('variables', {
+      variablesReference: variable.variablesReference
+    });
+    let newVariable = { ...variable };
+    reply.body.variables.forEach((ele: DebugProtocol.Variable) => {
+      newVariable = { [ele.evaluateName]: ele, ...newVariable };
```

Comment:
Should this be `ele.name` instead of `ele.evaluateName`? Otherwise attributes are displayed `a.foo` instead of `foo` for the following code sample:

```python
class A:
    def __init__(self):
        self.foo = 'foo'
        self.bar = 'bar'

a = A()
```

Reply:
Also we could rename `ele` to`variable` to be more explicit.